saml-example/docker-compose.yml

```diff
@@ -12,8 +12,6 @@ services:
       - MB_SAML_ATTRIBUTE_FIRSTNAME=firstName
       - MB_SAML_ATTRIBUTE_LASTNAME=lastName
       - MB_DB_FILE=/data/metabase.db
-    volumes:
-      - metabase-data:/data
```

Comment:
Why remove the data volume? IIRC it's so that you can use an existing app db, or retrieve the working one from within the Docker environment.

Reply:
I did this so the example does not leave a persistent H2 db in the environment that is running